vnext/Microsoft.ReactNative/Microsoft.ReactNative.vcxproj.filters

```diff
@@ -92,9 +92,6 @@
     <ClCompile Include="..\ReactUWP\Modules\AppThemeModuleUwp.cpp">
       <Filter>Modules</Filter>
     </ClCompile>
-    <ClCompile Include="..\ReactUWP\Modules\DeviceInfoModule.cpp">
-      <Filter>Modules</Filter>
```

Comment:
><Filter>Modules</Filter> [](start = 6, length = 24)

The PR doesn't contain a corresponding change to the .vcxproj. Is it not needed?

Reply:
It seems that VS has a bug managing the filters file. The file was removed by another PR, but when someone opens the project we see the filter file changed. I do not know a good way to avoid it unless VS fixes the issue.

Reply:
It is done by some other change. VS has a bug that filters file is not updated/saved correctly. When next developer opens VS project they see such random changes. No harm to bring this change along.

---
In reply to: [433407937](https://github.com/microsoft/react-native-windows/pull/5071#discussion_r433407937) [](ancestors = 433407937)